src/css/textarea.less

```diff
@@ -6,6 +6,11 @@
 
     // TODO: why is this here?
     .user-select(text);
+
+   > textarea {
+     opacity: 0; // Defense against iOS cursor appearing in client pages
```

Comment:
@karoltarasiuk, @laughinghan, yes it's the same issue. You can safely remove this line.